app/components/projects/settings/project_custom_field_sections/custom_field_row_component.html.erb

```diff
@@ -5,7 +5,7 @@
     }) do |custom_field_container|
       custom_field_container.with_column(flex_layout: true) do |title_container|
         title_container.with_column(pt: 1, mr: 2) do
-          render(Primer::Beta::Text.new) do
+          render(Primer::Beta::Text.new(classes: 'filter-target-visible-text')) do
```

Comment:
This removes a probably unintentional ability to search by type, by filtering for `string`, `bool`, …